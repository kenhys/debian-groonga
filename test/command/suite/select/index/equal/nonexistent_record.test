table_create Tags TABLE_PAT_KEY ShortText

table_create Bookmarks TABLE_HASH_KEY ShortText
column_create Bookmarks tag COLUMN_SCALAR Tags

column_create Tags index COLUMN_INDEX Bookmarks tag

load --table Bookmarks
[
{"_key": "http://groonga.org/", "tag": "groonga"},
{"_key": "http://mroonga.org/", "tag": ""}
]

select Bookmarks --query 'tag:\"nonexistent\"'
