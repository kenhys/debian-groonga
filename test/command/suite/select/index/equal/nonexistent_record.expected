table_create Tags TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
table_create Bookmarks TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Bookmarks tag COLUMN_SCALAR Tags
[[0,0.0,0.0],true]
column_create Tags index COLUMN_INDEX Bookmarks tag
[[0,0.0,0.0],true]
load --table Bookmarks
[
{"_key": "http://groonga.org/", "tag": "groonga"},
{"_key": "http://mroonga.org/", "tag": ""}
]
[[0,0.0,0.0],2]
select Bookmarks --query 'tag:\"nonexistent\"'
[[0,0.0,0.0],[[[0],[["_id","UInt32"],["_key","ShortText"],["tag","Tags"]]]]]
